solara/website/pages/__init__.py

```diff
@@ -251,18 +251,21 @@ def Layout(children=[]):
 
     target, set_target = solara.use_state(0)
 
-    if route_current and route_current.path == "apps":
-        return children[0]
-    if route_current is not None and route_current.path == "/":
-        return Home()
-    with solara.VBox(grow=False) as main:
-        Title(title="Solara documentation")
+    if route_current is not None:
         solara.Meta(name="twitter:card", content="summary_large_image")
         solara.Meta(name="twitter:site", content="@solara_dev")
         solara.Meta(name="twitter:image", content="https://solara.dev/static/assets/images/logo-small.png")
         solara.Meta(property="og:url", content="https://solara.dev" + router.path)
         solara.Meta(property="og:image", content="https://solara.dev/static/assets/images/logo-small.png")
         solara.Meta(property="og:type", content="website")
+    if route_current and route_current.path == "apps":
+        Title(title="Solara: Example Apps")
+        return children[0]
+    if route_current is not None and route_current.path == "/":
+        Title(title="Solara: Build high-quality web applications in pure Python")
+        return Home()
```

Comment:
Hmm, this won't work, because the return 'overrides' the auto collecting. Not sure if the layout messes up when we have it without the return (because we get a column around it).
Maybe 
```python
with solara.Div():
  Title(..)
  Home()
```

Reply:
Removing the return works for the most part, but we run into https://github.com/widgetti/solara/issues/626 in a particularly bad way.

Reply:
See https://github.com/widgetti/solara/commit/3ebea2da1abfc15e5b394849ca82e6a89b65f011 for a solution

Reply:
Works now, although the diff looks terrible.